components/terraform/Overview.vue

```diff
@@ -0,0 +1,107 @@
+                        <p>The provider is built on top of Kestra's API, allowing you to programmatically manage all resources, from workflows, namespaces, and secrets, to user management.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="hero row justify-content-center align-items-center">
+            <div class="col-12 col-md-6">
+                <img src="/terraform/overview/declarative-configurations-code.svg" class="w-100" alt="declarative configurations code svg">
+            </div>
+            <div class="col-12 col-md-6">
+                <div class="row justify-content-center">
+                    <div class="col-12 col-md-7">
+                        <h3>Declarative Configuration</h3>
+                        <p>Terraform uses its HashiCorp Configuration Language (HCL), while Kestra flows are defined in YAML. This alignment in declarative interfaces ensures not only a smooth transition but also makes it easy to use both technologies together.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-6">
+                <div class="row justify-content-center">
+                    <div class="col-12 col-md-7">
+                        <h3>Modularity in the Infrastructure Management</h3>
+                        <p>Kestra's Terraform provider is designed to easily manage all resources using Terraform. This includes, among others, flows, namespace, and user management, as well as secrets. Check the list of available resources in our <NuxtLink href="/docs/terraform" class="text-decoration-underline link-offset-2">documentation.</NuxtLink></p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-12 col-md-6">
+                <img src="/terraform/overview/modular-resource-management-code.svg" class="w-100" alt="modular resource management code svg">
+            </div>
+            <div class="col-12 col-md-6">
+                <p class="mt-5 text-white">The inherent modularity in both, Terraform and Kestra, allows you to create reusable blocks of code. These can be shared across multiple teams or projects, contributing to maintainable operations.</p>
+            </div>
+        </div>
+    </LayoutSection>
+</template>
+
+<style scoped lang="scss">
+@import "../../assets/styles/variable";
+.overview {
+    background: url('/terraform/overview/background.svg') no-repeat 0 -4%;
+}
+
+.card {
+    background-color: #110221;
+    color: white;
+    box-shadow: none !important;
+    
+    .card-body {
+        padding: 2rem 1rem;
+
+        h6 {
+            line-height: 1.875rem;
+        }
+
+        p {
+            font-size: $font-size-sm;
+            line-height: 1.375rem;
+            color: #CFCEFF;
+        }
+    }
+
+}
+
+.hero {
+    padding-bottom: 0;
+
+    p {
+        line-height: 1.5rem;
+        color: #CFCEFF;
```

Comment:
Add a variable, create one if necessary, so we can list all colors used in website